### DISCUSSION TITLE
Integrate controller with HomeAssistant to manually trigger boosting zones

### DISCUSSION THREAD

**Discussion by thwandi** [Beormund/Tasmota32-Multi-Zone-Heating-Controller#16] [Q&A]
Hello,

I just found this project because we are looking for a 3 zone heating standalone controller with a touchscreen. We are also using Automations in HomeAssistant to trigger our old system for boosting specific zones. Can the Zones Operating modes be triggered by Home Assistant (MQTT)? Right now we are triggering Automations with Zigbee buttons. They are switching on the 1hr boost of our zones and deactivating them after a timer (15, 30, 45 minutes) have passed (based on the time we have selected in our Automation).

**Comment by Beormund** [comment]
Yes. I do this in my setup. For example, to boost zone 1, have MQTT issue the command:

wifi2mqtt/Heating/cmnd/HeatingZone1 {"update": {"mode": 1, "hours": 1}}. This will boost zone 1 for 1 hour.

<img width="393" alt="Screenshot 2023-01-31 at 12 06 33" src="https://user-images.githubusercontent.com/75735592/215755723-f5beade2-6b5e-4e52-9c8b-8b6d783f71ee.png">

Check the "Commands" section of the readme documentation for a list of available commands.
https://github.com/Beormund/Tasmota32-Multi-Zone-Heating-Controller#commands
